ci/cirrus_wheels.yml

```diff
@@ -19,6 +19,7 @@ cirrus_wheels_linux_aarch64_task:
     platform: linux
     cpu: 2
     memory: 4G
+    timeout_in: 72m
```

Comment:
I tried restarting twice; this is probably cheaper than needing to hit restart several times during the rel process.

Reply:
+1 that makes sense to me, it was always close to the limit.